cmd/controller-manager/app/options/options.go

```diff
@@ -42,6 +45,7 @@ func NewControllerManagerServer() *ControllerManagerServer {
 			ContentType:                  "application/json",
 			K8sKubeconfigPath:            "./kubeconfig",
 			ServiceCatalogKubeconfigPath: "./service-catalog-kubeconfig",
+			ResyncInterval:               1 * time.Minute,
```

Comment:
I'd like all of these values to be `const`s, but for now can you put this new value into a `const`?